examples/jsm/loaders/LUTCubeLoader.js

```diff
@@ -44,80 +67,74 @@ export class LUTCubeLoader extends Loader {
 
 	}
 
-	parse( str ) {
+	parse( input ) {
 
-		// Remove empty lines and comments
-		str = str
-			.replace( /^#.*?(\n|\r)/gm, '' )
-			.replace( /^\s*?(\n|\r)/gm, '' )
-			.trim();
+		const regExpTitle = /TITLE +"([^"]*)"/;
+		const regExpSize = /LUT_3D_SIZE +(\d+)/;
+		const regExpDomainMin = /DOMAIN_MIN +([\d.]+) +([\d.]+) +([\d.]+)/;
+		const regExpDomainMax = /DOMAIN_MAX +([\d.]+) +([\d.]+) +([\d.]+)/;
+		const regExpDataPoints = /^([\d.e+-]+) +([\d.e+-]+) +([\d.e+-]+) *$/gm;
```

Comment:
Is using regular expressions to extract data actually faster than the previous approach (which just iterated once line by line through the entire file)?

Reply:
I tried checking execution times with `console.time/timeEnd` for the old and new loaders and got incosistent results. Both approaches take 5-20 milliseconds, but the `RegExp` approach appeared to be slightly faster in all cases.

Reply:
The motivation was to improve code clarity and to move the `LUT_3D_SIZE` and `DOMAIN_MIN/MAX` detection out of the main data loop.

Reply:
Thanks for the additional information! I'm okay with this new style however it would be good when @gkjohnson reviews this section as well since he is the original author of the both loaders (see #20558).